src/models/rating.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const raterSchema = Schema({
+  raterId: {
+    type: Schema.Types.ObjectId,
+    ref: process.env.USER_MODEL_NAME,
+    required: [true, 'Rater id is required'],
+  },
+  rating: {
+    type: Number,
+    min: 0,
+    max: 5,
+    required: [true, 'Rating value is required'],
+  },
+});
+
+const ratingSchema = Schema({
+  userRef: {
```

Comment:
It would be better to rename this to `user`. Just like the other 2 models have `owner` and `borrowers`. So that when you use the `populate()` feature of mongoose, `user` field will become an object will all the user information in the final result returned from the DB.